src/components/input/_macro.njk

```diff
@@ -24,13 +24,15 @@
             class="input input--text input-type__input {{ params.classes }}{{ exclusiveClass }}"
             {% if params.prefix or params.suffix %}title="{{ params.prefix.title if params.prefix }}{{ params.suffix.title if params.suffix }}"{% endif %}
             {% if params.name is defined %}name="{{ params.name }}"{% endif %}
+            {% if params.accessiblePlaceholder is defined %}required{% endif %}
             {% if params.value is defined %}value="{{ params.value }}"{% endif %}
             {% if params.accept is defined %}accept="{{ params.accept }}"{% endif %}
-            {% if params.min is defined %}min="{{ params.min }}"{% endif %}
-            {% if params.max is defined %}max="{{ params.max }}"{% endif %}
```

Comment:
Don't we need to keep these for e.g. date input?

Reply:
Yes. I'm not sure why I even changed it.